openshift/manifests/14-clustercatalog-catalogd-openshift-certified-operators.yml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: ClusterCatalog
+metadata:
+  name: catalogd-openshift-certified-operators
```

Comment:
Is there a way to prevent the `catalogd-` name prefix being added to these resources?

Reply:
In the case that we can't, I think something like `openshift-catalogd` would be better than `catalogd-openshift`

Reply:
Done in 7e87a7e1